scripts/linux/psv/travis_build_psv.sh

```diff
@@ -1,5 +1,11 @@
 #!/bin/bash -xe
-mkdir -p build && cd build
-cmake -DCMAKE_BUILD_TYPE=$BUILD_TYPE -EDGE_SDK_BUILD_EXAMPLES=ON -DBUILD_SHARED_LIBS=ON ..
+mkdir -p build
+cd build
+cmake -DCMAKE_BUILD_TYPE=$BUILD_TYPE \
+    -EDGE_SDK_BUILD_EXAMPLES=ON \
+    -DBUILD_SHARED_LIBS=ON \
+    -DCMAKE_CXX_COMPILER_LAUNCHER=ccache \
+    ..
+
 make -j8
 cd ..
```

Comment:
To prove that this really works please add `ccache -s`so that we see how many cache hits we had.